.github/README.md

```diff
@@ -5,7 +5,7 @@
 
 Collection of different ready to use reference architecture (RefArch) components developed by it@M.
 
-As counterpart there are also [refarch-templates](https://github.com/it-at-m/refarch-templates).
+For a new RefArch gitrepository use the [refarch-templates](https://github.com/it-at-m/refarch-templates).
```

Comment:
```suggestion
For creating a new RefArch project use the [refarch-templates](https://github.com/it-at-m/refarch-templates).
```

Reply:
Understand the problem.
I more thought about a software project (completely independent from github or gitlab).
Still think the newer formulation is more understandable. Didn't even think about github projects.